ydcommon/file_system_storage.py

```diff
@@ -0,0 +1,17 @@
+import os
+from django.contrib.staticfiles.storage import StaticFilesStorage
+from django.conf import settings
+
+
+class YDcommonFileSystemStorage(StaticFilesStorage):
+
+    def post_process(self, files, *args, **kwargs):
+        # print files
+        results = []
+        compress_images = getattr(settings, 'YDCOMMON_COMPRESS_STATIC_IMAGES', False)
+        if 'image_diet' in settings.INSTALLED_APPS and compress_images:
```

Comment:
And why? Image diet is user choice, he may not want to use it. In that case it should not fail.